### ISSUE TITLE
iOS 10.2 iPhone 6 crashes

### ISSUE THREAD

**Issue by 0xJacky** [angelXwind/PreferenceOrganizer2#29]
Hi,

I have installed the latest version 3.6, but it still crashes on my iPhone 6.

Crash Log:
```
Error	13:20:52.033691 +0800	SpringBoard	HW kbd: Failed to set (null) as keyboard focus
Error	13:20:52.085911 +0800	symptomsd	Attempt to add an app with insufficient id, info <private>
Error	13:20:52.175316 +0800	backboardd	Unable to bootstrap_look_up port with name .gsEvents: unknown error code (1102)
Error	13:20:52.186455 +0800	backboardd	Unable to bootstrap_look_up port with name .gsEvents: unknown error code (1102)
Error	13:20:52.195359 +0800	backboardd	Unable to bootstrap_look_up port with name .gsEvents: unknown error code (1102)
Error	13:20:52.226856 +0800	symptomsd	Can't get most elevated app state from dictionary <private>
```

**Comment by angelXwind** [comment]
Try downloading the latest version now, 3.7.

**Comment by 0xJacky** [comment]
Thanks
